mixin/package.go

```diff
@@ -103,7 +103,7 @@ func addPackage(pkg string, build bool) (string, error) {
 		return "", err
 	}
 	err = b.InitMix(fmt.Sprintf("%d", ver), fmt.Sprintf("%d", mixVer),
-		false, false, "https://download.clearlinux.org", false)
+		false, false, true, "https://download.clearlinux.org", false)
```

Comment:
Can you add "os-core" after this point and remove the `ioutil.WriteFile("mixbundles")` from `mixin/helpers.go:setUpMixDir` that was designed to avoid the defaultbundles additions?

Reply:
@matthewrsj  just pushed a new commit that I think is what you're asking for.